packages/legacy/core/App/container-api.ts

```diff
@@ -124,11 +148,18 @@ export type TokenMapping = {
   [TOKENS.CACHE_CRED_DEFS]: { did: string; id: string }[]
   [TOKENS.CACHE_SCHEMAS]: { did: string; id: string }[]
   [TOKENS.UTIL_LOGGER]: BaseLogger
-  [TOKENS.UTIL_OCA_RESOLVER]: OCABundleResolverType
+  [TOKENS.UTIL_OCA_RESOLVER]: RemoteOCABundleResolver
   [TOKENS.UTIL_LEDGERS]: IndyVdrPoolConfig[]
   [TOKENS.UTIL_PROOF_TEMPLATE]: ProofRequestTemplateFn | undefined
   [TOKENS.UTIL_ATTESTATION_MONITOR]: AttestationMonitor
   [TOKENS.FN_LOAD_HISTORY]: FN_HISTORY_MANAGER
+  [TOKENS.CONFIG]: Config
+  [TOKENS.FN_CRED_LIST_HEADER_RIGHT]: React.FC
```

Comment:
Why are some of these React FC's prefixed with `FN` and some prefixed with `COMPONENT`?